### ISSUE TITLE
FFMpeg through FFmpegWrapper

### ISSUE THREAD

**Issue by VicenteFava** [chrisballinger/FFmpeg-iOS#6]
I am using this library through this wrapper:

https://github.com/OpenWatch/FFmpegWrapper

However, I am getting the error: 

[NULL @ 0x7fb59a012a00] Unable to find a suitable output format for '/Users/mytestuser/Library/Developer/CoreSimulator/Devices/E0F92C98-411C-4CEA-8FC9-CE8E33F0A222/data/Containers/Data/Application/069294B7-7D45-4760-B3B2-030649DDFE1A/Documents/video'
2015-08-17 17:30:34.400 TestProject[9671:309324] codec not found: h264

What can I do to fix the issue?
Remember that I installed it all through cocoapods.

**Comment by toptierlabs** [comment]
Same issue here

**Comment by chrisballinger** [comment]
That's actually not an error. You should only use the H.264 hardware
encoder via VideoToolbox.framework.

**Comment by VicenteFava** [comment]
@chrisballinger How do I do that? I compiled ffmpeg with h264 codec. 
What should I change?

**Comment by demianardus** [comment]
@chrisballinger @VicenteFava I was having the same problem and i made the following change to FFOutputStream:

`AVCodec *codec = avcodec_find_encoder_by_name([outputCodec UTF8String]);`
For:
`AVCodec *codec = avcodec_find_encoder(AV_CODEC_ID_H264);`

Now that message is gone, and the codec is properly assigned, but I'm getting the following error when writing a packet into the file at this line:

` int writeFrameValue = av_interleaved_write_frame(self.formatContext, packet);`

Encoder did not produce proper pts, making some up.

Any ideas?

**Comment by VicenteFava** [comment]
I don't think that is the problem, the problem that I'm having with `av_interleaved_write_frame` is an arithmetic issue. 

In that line I have the following error

`EXC_ARITHMETIC(code=EXC_1386_DIV, subcode=0x0)`

I think it's because is traying to do a division by zero

**Comment by ziyuanliu** [comment]
+1

**Comment by fzakaria** [comment]
I've also built it with H264 and having the same 'cannot find codec' issue.
